resource/logging_macros.h.em

```diff
@@ -73,6 +80,9 @@ extern "C"
       condition_after \
     } \
   } while (0)
+#else
+#define RCUTILS_LOG_COND_NAMED(severity, condition_before, condition_after, name, ...)
+#endif //RCUTILS_NO_LOGGING
```

Comment:
Should we need to handle the rest of the macros?

Reply:
Well I'm not sure about this Python generated macro file functioning... but the other macros seems to be auxiliary. See fir example: 

```
#define RCUTILS_LOG_CONDITION_EXPRESSION_BEFORE(expression) \
  if (expression) {
```

In any case, by setting `RCUTILS_LOG_MIN_SEVERITY` to `NONE` here: https://github.com/micro-ROS/rcutils/blob/1a98061dcd5d9a8f0e88cab051292f279191021d/resource/logging_macros.h.em#L49 must disable the vast majority of macros.

The main problem is functions and static variables which I think that are disabled (errors and omission excepted).